bower.json

```diff
@@ -20,11 +20,11 @@
   "dependencies": {
     "n-ui-foundations": "^6.0.0",
     "next-session-client": "^3.0.1",
-    "o-buttons": "^6.0.2",
+    "o-buttons": "^6.0.0",
     "o-overlay": "^3.0.0",
-    "o-tracking": "^2.0.3",
+    "o-tracking": "^3.0.0",
     "o-visual-effects": "^3.0.0",
-    "o-message": "^4.1.8",
-    "o-icons": "^6.3.0"
+    "o-message": "^4.0.0",
+    "o-icons": "^6.0.0"
```

Comment:
Question: why do you downgrade these minors?

Reply:
just to keep it consistent. since the ^already patches to the most recent minor version.